lib/composables/useKResponsiveElement.js

```diff
@@ -5,11 +5,13 @@ let _resizeObserver;
 
 if (typeof window !== 'undefined' && window.ResizeObserver) {
   _resizeObserver = new ResizeObserver(entries => {
-    for (const entry of entries) {
-      if (entry.target._resizeListener) {
-        entry.target._resizeListener();
+    requestAnimationFrame(() => {
+      for (const entry of entries) {
+        if (entry.target._resizeListener) {
```

Comment:
Since there could be some time between when the resizer callback is fired and the animation frame callback is fired, I think it would be good to check the `entry.target`. In this case, the reference should still exist, but perhaps it isn't connected anymore: https://developer.mozilla.org/en-US/docs/Web/API/Node/isConnected 

Defensive code rarely hurts!